packages/fxa-content-server/app/scripts/views/mixins/signin-mixin.js

```diff
@@ -10,6 +10,7 @@ import NavigateBehavior from '../behaviors/navigate';
 import ResumeTokenMixin from './resume-token-mixin';
 import VerificationMethods from '../../lib/verification-methods';
 import VerificationReasons from '../../lib/verification-reasons';
+import Storage from '@fxa-shared/lib/storage';
```

Comment:
Note I'm using the fxa-shared version of this library and not the one available in the content server, for get/set consistency between the content and settings apps. One day we should point all storage lib references to fxa-shared.